test/test_jit_fuser_te.py

```diff
@@ -1815,7 +1815,12 @@ def fn(x):
                 self.assertEqual(len(self.findFusionGroups(t.graph_for(x))), 0)
 
     def test_superslomo(self):
-        for device in self.devices:
+        devices = []
```

Comment:
should we do 
```
devices = self.devices
```
if not RUN_CUDA and "cuda" in devices:
    devices.remove("cuda")
```

Reply:
Oh sure, that would work

Reply:
Ah, I think I need to do
```
devices = self.devices.copy()
```
or else this ends up mutating `self.devices`